README.md

```diff
@@ -96,25 +113,57 @@ fundamental differences in quality and design:
+* **Rich parsing** - [Regex][docs.transforms.regex_parser], [Grok][docs.transforms.grok_parser], and [more][urls.vector_parsing_transforms] allow for rich parsing.
+* **Context enrichment** - [Enrich data with environment context][urls.vector_enriching_transforms].
+
+#### Operations
+
+* **Configurable concurrency** - All CPU cores ([service][docs.roles.service]) or just one ([agent][docs.roles.agent]) via the [`--threads` flag][docs.process-management#starting].
+* **CI friendly** - [Config linting][docs.administration.validating], [dry runs][docs.administration.validating], and [unit tests][docs.guides.unit-testing] make Vector CI friendly.
+* **Custom DNS** - [Custom DNS][docs.global-options#dns_servers] makes service discovery possible.
 * **Hot reload** - [Reload configuration on the fly][docs.process-management#reloading] without disrupting data flow.
-* **Configurable concurrency** - All CPU cores (service) or just one (agent) via the [`--threads` flag][docs.process-management#starting].
-* **Programmable transforms** - [Lua][docs.transforms.lua], [Javascript (coming soon)][urls.pr_721], and [WASM (coming soon)][urls.issue_1802].
-* **Templating** - [Use event fields to create dynamic values][docs.reference.templating], enabling dynamic partitioning and more.
-* **Sink healthchecks** - Protection against downstream disruptions make Vector a good citizen.
-* **Customizable schema** - A schema is not forced on you. [Change Vector's schema][docs.global-options#log_schema] to anything you like.
+* **Optional static binary** - [Optional MUSL static binaries][pages.releases] mean zero required dependencies.
+* **Zero delay start** - Starts and restarts without a delay.
+* **Multi-platform** - [Linux, MacOS, Windows, x86_64, ARM64, and ARMv7][docs.installation].
+* **TLS support** - All relevant Vector components offer TLS options for secure communication.
+
+#### Reliability
+
 * **Decoupled buffer design** - Buffers are coupled with sinks; a bad sink won't bring the entire pipeline to a halt.
-* **Thoughtful disk buffering** - Transfers data to operating system memory to reduce the risk of data loss.
-* **Merge split logs** - Easily merge multi-line logs into one event.
-* **Custom DNS** - [Custom DNS][docs.global-options#dns_servers] makes service discovery possible.
-* **Rate-limited logging** - Vector's internal logging is rate-limited and noise-free.
-* **Optionally fully static binary** - Optional MUSL static binaries mean zero required dependencies.
-* **Thoughtful docs** - [Quality documentation][docs.what-is-vector] that respects your time and reduces communication overhead.
-* **Clear Guarantees** - A [guarantee support matrix][docs.guarantees] helps you make the appropriate tradeoffs.
+* **Fibnacci backoffs with jitter** - Vector's backoff algorithm makes Vector a good citizen during outages.
```

Comment:
Typo: "Fibnacci"

Might be easier to just say "Intelligent retries" or something like that.